src/bin/ion/commands/beta/generate/model.rs

```diff
@@ -167,16 +162,13 @@ pub struct WrappedScalar {
     // Corresponding `FullyQualifiedTypeReference` would be as following:
     // ```
     // FullyQualifiedTypeReference {
-    //    type_name: vec!["Foo"], // name of the top level scalar type
-    //    parameters: vec![FullyQualifiedTypeReference {type_name: vec!["String"] }] // nested type name for the scalar value
+    //    type_name: vec!["Foo"], // name of the wrapped scalar type
+    //    parameters: vec![FullyQualifiedTypeReference {type_name: vec!["String"] }] // base type name for the scalar value
     // }
     // ```
     name: FullyQualifiedTypeReference,
-    // Represents the scalar type
     // Represents doc comment for the generated code
-    // If this is a top level scalar type definition then this will have `Some(doc_comment)`,
-    // Otherwise this is `None`.
-    doc_comment: Option<String>,
+    doc_comment: String,
```

Comment:
Any particular reason this is no longer `Option`? I think it's still possible for it to be `None`. It's just that it doesn't depend on whether this is top-level or not.

Reply:
Hmm. That makes sense. Will change that.